spring-cloud-azure-autoconfigure/src/main/java/com/microsoft/azure/spring/cloud/autoconfigure/cosmosdb/AzureCosmosDbMongodbProperties.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.cloud.autoconfigure.cosmosdb;
+
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.util.Assert;
+import javax.annotation.PostConstruct;
+
+@Getter
+@Setter
+@ConfigurationProperties("spring.cloud.azure.cosmosdb.mongodb")
+public class AzureCosmosDbMongodbProperties {
```

Comment:
I think mongo and cansandra might share one common `AzureCosmosDbPropreties`

Reply:
I think so. I'll change this class name.